docs/conf.py

```diff
@@ -50,7 +50,11 @@
     },
 }
 jinja_filters = {
-    'sort_engines': lambda engines: sorted(engines, key=lambda engine: (engine[1].disabled, engine[0]))
+    'sort_engines':
+    lambda engines: sorted(
+        engines,
+        key=lambda engine: (engine[1].disabled, engine[1].about.get('language', ''), engine[0])
```

Comment:
Nit: can you add a space left & right to the equation symbol / even in arguments / I know it is not PEP8, but it is much better to read ;)

    key = lambda engine: (...)

Reply:
I'm happy to adhere to any formatting style you choose for your project. I'm however also a fan of consistency and currently the code mostly sticks to the PEP8 style. Given that there's a recent PR to introduce auto-formatting #619, applying selective manual formatting here seems weird to me.

Reply:
> I'm happy to adhere to any formatting style you choose for your project. I'm however also a fan of consistency and currently the code mostly sticks to the PEP8 style. 

Our code is (mostly) PEP8 / but not in this one case: What PEP8 says about formatting arguments is good for on-liners but not for a multiline argument list, especial not where the value of an argument takes several lines (IMO PEP8 is pointless in this case).

> Given that there's a recent PR to introduce auto-formatting ..

Yes we have #619 but this PR is in a draft state and I don't know when we have finished a code-style workflow for python like we have done this month for JS.

```
jinja_filters = {
    'sort_engines': lambda engines: sorted(
        engines,
        key = lambda engine: (
            engine[1].disabled, engine[1].about.get('language', ''), engine[0]
        ),
    )
}
```

.. is much better to read / nevertheless, what PEP8 says .. AFAIK black won't fix this to PEP8 style.

Reply:
```
$ grep -Pzohr '[(,][\n\t ]*[a-z]+=[^\n]+' --include='*.py' searx  | tr "$'\0'" $'\n' | grep = | wc -l
413
$ grep -Pzohr '[(,][\n\t ]*[a-z]+ =[^\n]+' --include='*.py' searx  | tr "$'\0'" $'\n' | grep = | wc -l
77
```

The PEP8 style seems way more common in the codebase.

>AFAIK black won't fix this to PEP8 style

Actually it does.

Reply:
> Actually it does.

Seems you are right .. I will never understand why this arbitrary setting of whitespaces ..

```
dict_a = {
    "foo": "bar",
    "john": "doe",
}

dict_b = dict(
    foo="bar",
    john="doe",
)
```

should be more readable than ..

```
dict_a = {
    "foo" : "bar",
    "john" : "doe",
}

dict_b = dict(
    foo = "bar",
    john = "doe",
)
```
.. anyway if the majority thinks every detail of PEP8 weights more than readability ..